src/pkg/build/assemblers/assembler_sif.go

```diff
@@ -81,6 +83,11 @@ func createSIF(path string, definition []byte, squashfile string) (err error) {
 		return fmt.Errorf("while creating container: %s", err)
 	}
 
+	// chown the sif file to the calling user
+	if uid, gid, ok := changeOwner(); ok {
+		os.Chown(path, uid, gid)
```

Comment:
We should always be checking for errors from functions which return errors

Reply:
agree.
done in ec1490d